java/code/src/com/redhat/rhn/domain/kickstart/KickstartInstallType.java

```diff
@@ -102,39 +100,11 @@ public boolean isRhel6() {
         return RHEL_6.equals(getLabel());
     }
 
-    /**
-     * @return true if the installer type is rhel 5
-     */
-    public boolean isRhel5() {
-        return RHEL_5.equals(getLabel());
-    }
-
-    /**
-     * @return true if the installer type is rhel 4
-     */
-    public boolean isRhel4() {
-        return RHEL_4.equals(getLabel());
-    }
-
-    /**
-     * @return true if the installer type is rhel 53
-     */
-    public boolean isRhel3() {
-        return RHEL_3.equals(getLabel());
-    }
-
-    /**
-     * @return true if the installer type is rhel 2
-     */
-    public boolean isRhel2() {
-        return RHEL_21.equals(getLabel());
-    }
-
     /**
      * @return true if the installer type is rhel
      */
     public boolean isRhel() {
-        return isRhel2() || isRhel3() || isRhel4() || isRhel5() || isRhel6() || isRhel7() || isRhel8();
+        return isRhel6() || isRhel7() || isRhel8();
```

Comment:
Missing `isRhel9()` and also its implementation.

Reply:
thanks. Adding it